chartfx-samples/src/main/java/de/gsi/chart/samples/financial/service/execution/BacktestExecutionPlatform.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * LGPL-3.0, 2020/21, GSI-CS-CO/Chart-fx, BTA HF OpenSource Java-FX Branch, Financial Charts
+ */

```

Comment:
remove, please.

There seems to be a lot of financial non-UI domain functionality. Should one consider refactoring this into another dedicated sub-module, like 'dataset-financial'?